<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>190</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="DCDC_ID_label">
      <property name="text">
       <string>DCDC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dcdc_id_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>42</y>
     <width>373</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>output_vol</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="get_output_vol_lineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="set_output_vol_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="showplainTextEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="startpushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>input file:</string>
   </property>
  </widget>
  <widget class="QTableView" name="m_tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>691</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="timelineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
